pkg/volume/util/operationexecutor/operation_generator.go

```diff
@@ -277,17 +277,26 @@ func (og *operationGenerator) GenerateAttachVolumeFunc(
 				volumeToAttach.VolumeSpec.Name(),
 				volumeToAttach.NodeName,
 				attachErr)
-			for _, pod := range volumeToAttach.ScheduledPods {
-				og.recorder.Eventf(pod, v1.EventTypeWarning, kevents.FailedMountVolume, err.Error())
```

Comment:
Maybe users still want to see such events of their pods in order to know this failure as details?